Jenkinsfile

```diff
@@ -83,21 +83,30 @@ pipeline {
             }
             }
         }
-        stage("pytest3 and Tests Coverage"){
-            agent { label utils.rocmnode("tunatest") }
+        stage("pytest3"){
+            agent{  label "gfx90a" }
             steps {
-                script {
-                    utils.pytestSuite3AndCoverage(branch, branch_master)
-                }
+            script {
+            utils.pytestSuite3()
+            }
             }
         }
-        stage("fin find compile"){
-        agent{ label utils.rocmnode("tunatest") }
-        steps{
+        /*
+        stage("Coverage"){
```

Comment:
Might need to un-comment or remove this?

Reply:
i have this in a different PR where i made lots of coverage updates, so i left i commented out here so i dont pollute this PR with more changes